pubsub.go

```diff
@@ -8,4 +8,12 @@ type PubsubConfig struct {
 	// DisableSigning disables message signing. Message signing is *enabled*
 	// by default.
 	DisableSigning bool
+
+	// Enable pubsub (--enable-pubsub-experiment)
+	Enabled bool
```

Comment:
This should be the Flag type (https://github.com/ipfs/go-ipfs-config/blob/a412831517bb5ce3672af95e0b63c3633218e80e/types.go#L50) so that we can easily flip pubsub on by default when it's ready.